app/src/main/java/pl/droidsonroids/droidsmap/feature/employee/business_logic/EmployeeEntityHolder.kt

```diff
@@ -0,0 +1,7 @@
+package pl.droidsonroids.droidsmap.feature.employee.business_logic
+
+
+data class EmployeeEntityHolder (
+    val employee : EmployeeEntity,
```

Comment:
Wydaje mi się, że to nie zostało poddane automatycznej refaktoryzacji stylu - przed dwukropkiem nie powinno być chyba spacji